### ISSUE TITLE
Resizable answers

### ISSUE THREAD

**Issue by Pliskin-Hunter** [Macronair/TheMillionaireGame#64]
Hi there, thanks for this very complete software !

I just have a single issue where the answer is longer than few words, like:
"Michael Biehn's head on Brandon Lee's body" gets truncated to "Michael Biehn's head on"

Which doesn't really work out...
Making a fix for that would really help a lot doing better questions.

Thanks !

**Comment by jscmidt** [comment]
Yes, I'm also accounting errors when editing questions with levels
